libs/application/templates/health-insurance/src/forms/HealthInsuranceForm.ts

```diff
@@ -147,17 +150,21 @@ export const HealthInsuranceForm: Form = buildForm({
               width: 'half',
               disabled: true,
               defaultValue: (application: Application) =>
-                (application.externalData.nationalRegistry?.data as {
+                ((application.externalData.nationalRegistry?.data as {
+                  address?: Address
+                }).address as {
                   postalCode?: string
-                })?.postalCode,
+                })?.postalCode || '',
             }),
             buildTextField({
               id: 'applicant.city',
               title: m.city,
               width: 'half',
               disabled: true,
               defaultValue: (application: Application) =>
-                (application.externalData.nationalRegistry?.data as {
+                ((application.externalData.nationalRegistry?.data as {
+                  address?: Address
+                }).address as {
                   city?: string
                 })?.city,
```

Comment:
I don't know if you we need to define the type for city if address already has a type that contains city. Same with postalCode and streetAddress. Will typescript "complain" anyway?
What I mean is, does it work if we do like this:
`(application.externalData.nationalRegistry?.data as {
    address?: Address
}).address?.city`

Reply:
Missed that, will fix 👍